### PR TITLE
Connectivity check timout terminology fix

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-ice-pac.xml [hunk 1]
```diff
@@ -66,27 +66,27 @@
         for Network Address Translator (NAT) traversal for UDP-based communication.
       </t>
       <t>
-        Typically, when using ICE, endpoints will exchange candidate addresses,
+        Typically, when using ICE, ICE agent will exchange candidate addresses,
```
Comment (nils-ohlmeier) [inline draft-ietf-ice-pac.xml]: Plural - 'ICE agents'?

**Review by nils-ohlmeier — APPROVED** [review]
Looks good me, with one small improvement.

File: draft-ietf-ice-pac.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml]: Yes - will fix :)

**Comment by juberti** [comment]
RFC 8445 uses 'endpoint' more often than 'peer agent', and does not even officially define the term 'peer agent'. I'm not sure it makes sense to remove every single usage of 'implementation' or 'endpoint'.

File: draft-ietf-ice-pac.xml [hunk 2]
```diff
@@ -116,56 +116,56 @@
           candidates are very likely to result in a valid pair.
         </t>
         <t>
-          In most ICE scenarios, the lengthy timeouts for connectivity checks,
```
Comment (juberti) [inline draft-ietf-ice-pac.xml:119]: I would just say "for connectivity check transactions"

File: draft-ietf-ice-pac.xml [hunk 3]
```diff
@@ -116,56 +116,56 @@
...
+            checklist at the peer agent will once again be empty, leading
             to immediate ICE failure.
           </t>
         </section>
         <section title="Immediate Candidate Pair Failure">
           <t>
             Section 7.2.5.2 of <xref target="RFC8445"></xref> describes several
             situations in which a candidate pair will be considered to have
-            failed, well before the connectivity check timeout.
+            failed, well before the connectivity check Binding request transaction timeout.
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]: "connectivity check transaction timeout"

File: draft-ietf-ice-pac.xml [hunk 4]
```diff
@@ -177,22 +177,22 @@
         needs to wait enough time to allow peer-reflexive candidates to be
         discovered. Accordingly, when full ICE implementations begin their
         ICE processing, as described in <xref target="RFC8445" />, Section
-        6.1, the ICE agent MUST set a timer, and its duration SHOULD be
-        equal to the ICE agent's connectivity check timeout.
+        6.1, the agent MUST set a timer, and its duration SHOULD be
+        equal to the agent's Binding request transaction timeout.
```
Comment (juberti) [inline draft-ietf-ice-pac.xml]: "connectivity check transaction timeout"

File: draft-ietf-ice-pac.xml (hunk 3 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml]: I can do that.

File: draft-ietf-ice-pac.xml (hunk 4 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml]: I can do that.

File: draft-ietf-ice-pac.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-ice-pac.xml:119]: I can do that.

**Review by cdh4u — COMMENTED** [review]
Doh! I had copy/pasted the text there for reference, but forgot to remove it :)

**Comment by juberti** [comment]
Why did this PR get merged? See commentary above regarding terminology.

**Comment by cdh4u** [comment]
> Why did this PR get merged? See commentary above regarding terminology.

Sorry, I forgot that. My issue was not really "endpoint" versus "ICE agent" - both are used in RFC 8445. My issue was more regarding "remote", which I changed to "peer". I did change "endpoint" and "implementation" (there was only one instance) at the same time, but that was just because I wanted consistent terminology. 

Also, I didn't want to use "ICE agent" all over, so I used in on first occurance within a section, and then "agent" after that. But, if you think "peer ICE agent" would be more clear we could use that.

**Comment by juberti** [comment]
I read through the changes and there were a few places where I think the new terminology was less clear (e.g., where 'agent' could refer to either the local or peer agent), so I reverted the change. I'll prepare a new PR where we can discuss the details.

**Pull request merged**